packages/xo-web/src/xo-app/jobs/new/index.js

```diff
@@ -38,6 +40,56 @@ import {
 
 const JOB_KEY = 'genericTask'
 
+const COLUMNS = [
+  {
+    itemRenderer: job => (
+      <span>
+        {job.name} <span className='text-muted'>({job.id.slice(4, 8)})</span>
+      </span>
+    ),
+    name: _('jobName'),
+    sortCriteria: 'name',
+  },
+  {
+    itemRenderer: job => job.method,
+    name: _('jobAction'),
+    sortCriteria: 'method',
+  },
+  {
+    itemRenderer: (job, userData) => {
+      const { id } = job
+      const isJobUserMissing = userData.isJobUserMissing[id]
+
+      return (
+        <div>
+          <ActionRowButton
+            disabled={!isJobUserMissing}
+            icon='run-schedule'
+            btnStyle='warning'
+            handler={runJob}
+            handlerParam={id}
+          />
+          {!isJobUserMissing && (
+            <Tooltip content={_('jobUserNotFound')}>
+              <Icon className='ml-1' icon='error' />
+            </Tooltip>
+          )}
+        </div>
+      )
+    },
+  },
+]
+
+const GROUPED_ACTIONS = [
+  {
+    handler: deleteJobs,
+    handlerParam: selectedJobs => map(selectedJobs, 'id'),
```

Comment:
I think it would be better to have a transform function that only takes one item (`selectedJob => selectedJob.id`) and let the `GroupedAction` do the `map`. Then it would easily work with `actions`.